elastic/logs/templates/component/track-shared-logsdb-mode.json

```diff
@@ -0,0 +1,22 @@
+{
+    "template": {
+      "settings": {
+        "index": {
+          {% if index_sorting == "hostname" %}
+            "sort.field": [ "host.name", "@timestamp" ],
+            "sort.order": [ "asc", "desc" ]
+          {%- elif index_sorting == "timestamp" %}
+            "sort.field": [ "@timestamp", "host.name" ],
+            "sort.order": [ "desc", "asc" ]
+          {%- endif %}
+        }
+      },
+      "mappings": {
+        {% if index_sorting %}
```

Comment:
Nit: it's a bit awkward that index sorting enables synthetic source, is there any downside with configuring it with another parameter?

Reply:
There is no downside. But I did this with the index.mode=logs in mind. I think this would both enable index sorting and synthetic source. But I can add a synthetic source parameter for now, which would give us more benchmarking flexibility.